tests/unit/controllers/administrative-units/administrative-unit-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
```

Comment:
I would delete the test files. We don't write tests anyway :smile: and they sometimes cause false positives during CI.

Reply:
(I created a PR to delete the other ones already)

Reply:
I think there are still 2 test files remaining for the routes?

Reply:
Done!